api/v1/openapi.yaml

```diff
@@ -2509,6 +2509,14 @@ definitions:
           Indicator if this backend is preferred in the context of clustermesh service affinity. The value is set based
           on related annotation of global service. Applicable for active state only.
         type: boolean
+  BackendWeight:
```

Comment:
This seems a bit awkward -- is there a reason this is not a property on the BackendAddress object?

Reply:
this was proposed by the initial author, but makes more sense to me to be part of `BackendAddress` object :+1:

Reply:
let me fix that and the logic around that

Reply:
removed this and made it part of BackendAdress object